bin/delayed_job

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
```

Comment:
Are you sure this is necessary?

Reply:
This point still stands — I don't think you need any manual binstub when you have raketask, which works nicely from your Procfile